python/tests/plugins/validators/test__validate_usecase_participants.py

```diff
@@ -13,8 +13,8 @@
 
 class TestUsecaseParticipants(ActiveContextTestCase):
     def test_validate_usecase_with_referenced_participants(self):
-        test_participant_1 = create_field_entry("user1", "SomeModel")
-        test_participant_2 = create_field_entry("user2", "SomeOtherModel")
+        test_participant_1 = create_field_entry("user1", description="SomeModel")
+        test_participant_2 = create_field_entry("user2", description="SomeOtherModel")
```

Comment:
Based on how these have historically been used, they just follow the constructor order. Since no modifications were made to the `create_field_entry()`, think it should be okay to not push this to `description`.

```suggestion
        test_participant_1 = create_field_entry("user1", "SomeModel")
        test_participant_2 = create_field_entry("user2", "SomeOtherModel")
```